### PR TITLE
create x/superfluid/types/times.go with constants for durations

### PR DESCRIPTION
**Pull request by faddat**
Closes: #849

## Description

This removes the manually set times in the superfluid staking with constants in the types package. in a file called times.go

______

For contributor use:

- [x] Targeted PR against correct branch (see [CONTRIBUTING.md](https://github.com/cosmos/gaia/blob/main/CONTRIBUTING.md#pr-targeting))
- [x] Updated relevant documentation (`docs/`) or specification (`x/<module>/spec/`)
- [x] Added a relevant changelog entry to the `Unreleased` section in `CHANGELOG.md`
- [x] Re-reviewed `Files changed` in the Github PR explorer

### CODE REVIEW

File: x/superfluid/types/times.go [hunk 1]
```diff
@@ -0,0 +1,8 @@
+package types
+
+import (
+	"time"
+)
+
+const Twoweeks = time.Hour * 24 * 14
```
Comment (ValarDragon) [inline x/superfluid/types/times.go:7]: Lets rename to OsmosisMainnetUnbondingPeriod ?

File: x/superfluid/types/params.go [hunk 2]
```diff
@@ -34,7 +33,7 @@ func DefaultParams() Params {
 	return Params{
 		RefreshEpochIdentifier: "day",
 		MinimumRiskFactor:      sdk.NewDecWithPrec(5, 2), // 5%
-		UnbondingDuration:      time.Hour * 24 * 21,      // 3w
+		UnbondingDuration:      Threeweeks,
```
Comment (ValarDragon) [inline x/superfluid/types/params.go:36]: Lets switch to two weeks?

**Review by ValarDragon — APPROVED** [review]
ty!

File: x/superfluid/types/times.go (hunk 1 above)
Comment (faddat) [reply x/superfluid/types/times.go:7]: heh, I thought of this :)!

sure thing

File: x/superfluid/types/params.go (hunk 2 above)
Comment (faddat) [reply x/superfluid/types/params.go:36]: yep

Comment (faddat) [reply x/superfluid/types/params.go:36]: I thought that it was three because of external unbondings or nah?

Comment (ValarDragon) [reply x/superfluid/types/params.go]: Nah, it was an accident from it being written

**Comment by ValarDragon** [comment]
Can you fix the tests, then we can get this merged?

**Comment by ValarDragon** [comment]
Going to remove the need for the constants entirely.

**Pull request closed without merging**